webpack/scenes/RedHatRepositories/components/SearchBar.js

```diff
@@ -97,6 +97,7 @@ class SearchBar extends Component {
         <MultiSelect
           value={this.state.filters}
           options={filterOptions}
+          noneSelectedText={__("Filter by repo type")}
```

Comment:
Would it be too long to spell out 'repository' vs 'repo'?  It may be more consistent and easier for translations.

Reply:
Let's just go with "Filter by type".